src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -5120,6 +5120,73 @@ public static void Main()
             var verifier = CompileAndVerify(src, expectedOutput: @"1");
         }
 
+        [Fact]
+        public void BaseArguments_17()
```

Comment:
Is this test's or `BaseArguments_18` behavior modified by the code changes?

Reply:
> Is this test's or BaseArguments_18 behavior modified by the code changes?

No

---
In reply to: [437654044](https://github.com/dotnet/roslyn/pull/44992#discussion_r437654044) [](ancestors = 437654044)